README.md

```diff
@@ -56,7 +58,13 @@ cargo test # you *should* see errors like those below here...
 # 
 # test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
 ```
-6. Edit the code in the exercise folder and run `cargo test` again, when the tests pass you will see output that give a postitive results like the following:
+6. Edit the code in the exercise folder, e.g. by typing the following command in VS Code's [integrated terminal](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_integrated-terminal) (which can be opened with the [VS Code Shortcut](https://code.visualstudio.com/docs/getstarted/tips-and-tricks) `` Ctrl+` ``):
+
+
+after you have made some edits run `cargo test` again. **What if you get stuck, as everyone does at some point?**
+  - Don't worry: the error messages should help you learn how Rust works
+  - Don't suffer in silence! Feel free to ask in the issue tracker or in the Rust discord group for help: https://discord.gg/FRRenrPUSA [![discord](https://img.shields.io/discord/598002550221963289?label=discord&logo=Discord&color=blue)](https://discord.gg/Te3gWeDwmf)
```

Comment:
Slack invite links expire every so often, and I think Discord ones too (based on people pinging me for a refreshed link to another server). But this is a start! Someone can raise an issue if it becomes broken